<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Excel To PSQL</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="load_dataButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>420</y>
      <width>136</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить данные</string>
    </property>
   </widget>
   <widget class="QFrame" name="choose_columns_frame">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>411</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="pereEdit">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>201</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="columns_chooseLayout">
      <item>
       <widget class="QRadioButton" name="diapasonButton">
        <property name="text">
         <string>Диапазон:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="pereButton">
        <property name="text">
         <string>Перечисление: n,n1,n2...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="fromBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>по</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="toBox"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="choose_columnsButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>150</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Готово</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Выбор данных</string>
     </property>
    </widget>
    <widget class="QComboBox" name="c_sheetBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>37</y>
       <width>67</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Лист:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="del_spButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить строки с пробелами</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Connection_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>671</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="save_setButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>90</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить настройки</string>
     </property>
    </widget>
    <widget class="QLabel" name="passLabel">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>40</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="unLabel">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>40</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="unLine">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>60</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="load_setButton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>90</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Загрузить настройки</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dbnLine">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="portLabel">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Host Port</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connectButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Подключиться</string>
     </property>
    </widget>
    <widget class="QLabel" name="dbnLabel">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>DB Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passLine">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>60</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="hostLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Host IP</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hostLine">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="portLine">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Подключение</string>
     </property>
    </widget>
    <widget class="QComboBox" name="choose_tableBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="edit_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>251</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Редактирование</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_excelButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>219</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить файл</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ofButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>219</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Открыть файл</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="status_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>671</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="status_listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>651</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="status_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Статус:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
